content/c/concepts/linked-list/linked-list.md

```diff
@@ -0,0 +1,329 @@
+---
+Title: 'Linked List'
+Description: 'A data structure that contains a collection of nodes that together with the help of pointers, display a sequence.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Singly linked lists'
+  - 'Doubly linked lists'  
+  - 'Circular linked lists'
+  - 'Circular doubly linked lists'
+CatalogContent:
+  - 'learn-c'
+  - 'paths/computer-science'
+---
+
+A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are several list types, we will cover four of the most common with examples in the [C language](https://www.codecademy.com/resources/docs/c).
```

Comment:
```suggestion
A **`Linked List`** is a data structure that contains a collection of `nodes` that together with the help of pointers, display a sequence. There are several list types, four of the most common are discussed below.
```
Avoid first and second person pronouns in docs, also this is a C entry, there's no need to reiterate the examples are in C.

Reply:
This change has been update and pushed for review.